tensorflow/compiler/tf2tensorrt/convert/convert_nodes.h

```diff
@@ -158,7 +158,8 @@ Status ConvertGraphDefToEngine(
     TrtUniquePtrType<nvinfer1::ICudaEngine>* engine, bool use_calibration,
     const bool use_implicit_batch, bool* convert_successfully,
     TrtShapeOptimizationProfile* profiles, absl::string_view engine_name,
-    bool use_explicit_precision);
+    bool use_explicit_precision,
+    tensorflow::grappler::Cluster *cluster);
```

Comment:
We can make cluster=nullptr as default to avoid the change to the test.

Let's add this to the comment block for this routine:
When 'cluster' is not null, it contains the graph to be converted. We may perform additional optimizations to the graph before converting the graph.